src/libraries/Microsoft.Extensions.FileProviders.Composite/tests/MockFileProvider.cs

```diff
@@ -11,6 +11,13 @@ namespace Microsoft.Extensions.FileProviders.Composite
 {
     public class MockFileProvider : IFileProvider
     {
+#if NETCOREAPP
+        static MockFileProvider ()
+        {
+            Castle.DynamicProxy.Generators.AttributesToAvoidReplicating.Add(typeof(System.Diagnostics.CodeAnalysis.DynamicDependencyAttribute));
```

Comment:
Why is this needed? It may be useful to add comment.

Reply:
Comment added